QUEST_LV_0100.tsv

```diff
@@ -4723,15 +4723,15 @@ QUEST_LV_0100_20150317_004722	Pour the sources of the magic of the Mausoleum to
 QUEST_LV_0100_20150317_004723	The protector of the secret told you that the Mausoleum of Zachariel was actually made to defeat Rexipher according to the forecast by the Goddess. Recharge the sources of the magic energy of the Mausoleum to the prepared pot.
 QUEST_LV_0100_20150317_004724	You have charged all the sources of the magic energy to all pots. Talk to the protector of the secret.
 QUEST_LV_0100_20150317_004725	You felt like you've heard the sound of cries from the demons. Talk to the protector of the secret.
-QUEST_LV_0100_20150317_004726	{memo X}Lure Medakia into the pot and absorb
+QUEST_LV_0100_20150317_004726	{memo X}Lure and absorb Medakia into the pot
 QUEST_LV_0100_20150317_004727	You should deal a fatal blow to Rexipher with the wills of the protectors. Install the complete pots of the magic energy and lure them into the pots to fill them in.
 QUEST_LV_0100_20150317_004728	You filled the wills of the protectors into the pots. Talk to the protector of the secret.
```

Comment:
protectors -> guardian
Talk to the protector of the secret. -> Talk to the secret guardian